packages/eas-cli/src/graphql/mutations/EnvironmentSecretMutation.ts

```diff
@@ -0,0 +1,91 @@
+import { print } from 'graphql';
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import {
+  CreateEnvironmentSecretForAccountMutation,
+  CreateEnvironmentSecretForAppMutation,
+  DeleteEnvironmentSecretMutation,
+  EnvironmentSecretFragment,
+} from '../generated';
+import { EnvironmentSecretFragmentNode } from '../types/EnvironmentSecret';
+
+export const EnvironmentSecretMutation = {
+  async createForAccount(
+    input: { name: string; value: string },
+    accountId: string
+  ): Promise<EnvironmentSecretFragment> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .mutation<CreateEnvironmentSecretForAccountMutation>(
+          gql`
+            mutation CreateEnvironmentSecretForAccount(
+              $input: CreateEnvironmentSecretInput
```

Comment:
Maybe we should make the input required in our API?
I can see it's not required https://github.com/expo/universe/blob/main/server/www/src/graphql/mutations/EnvironmentSecretMutation.ts#L27
but then it's treated as if it always exists - https://github.com/expo/universe/blob/main/server/www/src/graphql/mutations/EnvironmentSecretMutation.ts#L58

Reply:
The same applies to `createEnvironmentSecretForApp`

Reply:
ah yeah, thanks for noticing this! it was definitely intended to be required